web/src/ee/features/billing/server/stripeWebhookApiHandler.ts

```diff
@@ -0,0 +1,311 @@
+import {
+  getOrgIdFromStripeClientReference,
+  isStripeClientReferenceFromCurrentCloudRegion,
+} from "@/src/ee/features/billing/stripeClientReference";
+import { env } from "@/src/env.mjs";
+import { type NextRequest, NextResponse } from "next/server";
+import { prisma } from "@langfuse/shared/src/db";
+import * as Sentry from "@sentry/node";
+import { stripeClient } from "@/src/ee/features/billing/utils/stripe";
+import type Stripe from "stripe";
+import { CloudConfigSchema, parseDbOrg } from "@langfuse/shared";
+
+const STRIPE_WEBHOOK_SIGNING_SECRET_DEV: string | null =
+  "whsec_12dc385262f1a5d0f4ba1507cc81f9b3a3e2d03fd99d4ad625b8e21c87dcfd37";
```

Comment:
looks like dev but should this be in here?

Reply:
yes, will be removed before going live, is non critical (dev mode only)